popout.js

```diff
@@ -1,42 +1,44 @@
 class PopoutModule {
 	static onRenderJournalSheet(obj, html, data) {
-		let element = html.find(".window-header .window-title")
-		PopoutModule.addPopout(element, `game.journal.get("${obj.entity.id}").sheet`);
+		let element = html.find(".window-header .window-title");
+		let sheetID = obj.entity.id;
+		PopoutModule.addPopout(element, `game.journal.get("${obj.entity.id}").sheet`, sheetID);
 	}
 	static onRenderActorSheet(obj, html, data) {
-		let element = html.find(".window-header .window-title")
-		PopoutModule.addPopout(element, `game.actors.get("${obj.entity.id}").sheet`);
+		let element = html.find(".window-header .window-title");
+		let sheetID = obj.entity.id;
+		PopoutModule.addPopout(element, `game.actors.get("${obj.entity.id}").sheet`, sheetID);
 	}
-	static addPopout(element, sheet) {
+	static addPopout(element, sheet, sheetID) {
 		// Can't find it?
 		if (element.length != 1) {
 			return;
 		}
 		let popout = $('<a class="popout" style><i class="fas fa-external-link-alt"></i>PopOut!</a>')
-		popout.on('click', (event) => PopoutModule.onPopoutClicked(event, sheet))
+		popout.on('click', (event) => PopoutModule.onPopoutClicked(event, sheet, sheetID))
 		element.after(popout)
 
 	}
 	static onPopoutClicked(event, sheet) {
-
 		// Check if popout in Electron window
 		if (navigator.userAgent.toLowerCase().indexOf(" electron/") !== -1) {
 			return ui.notifications.warn("Popout! cannot work within the standalone FVTT Application. Please open your game from a regular browser.");
 		}
-
-		let div = $(event.target).closest("div")
-		let window_title = div.find(".window-title").text().trim()
-
+		
+		let div = $(event.target).closest("div");
+		let window_title = div.find(".window-title").text().trim();
+		let obj = (game.actors.get(sheetID)) ? game.actors.get(sheetID) : game.journal.get(sheetID);

```

Comment:
Yeah, trying to guess between actor or journal by testing both with the same id, is not a good idea, better to receive `obj` directly via argument.